ShapeVariationAnalyzer/cpns/cpns.py

```diff
@@ -53,62 +55,165 @@ def __init__(self):
         self.PNSSpoke = None
         self.ZComp = None
 
+        # if our sreps are the new style, clone the first one so
+        # we have something to update spoke meshes on for saving.
+        # Because CPNS only cares about the sreps as if they were a point
+        # cloud, the underlying srep structure doesn't matter, and we don't
+        # have to know if it is a vtkEllipticalSRep or some other kind
+        self._newStyleSRepClone = None
+
+    def _UpdateSRepXML(self, filePath):
```

Comment:
Add docstring summarizing what this function does

Reply:
Added docstring.